.github/workflows/dockerhub.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         with:
           context: ./carma-messenger-config/chevrolet_tahoe_2018/
           push: true
-          tags: usdotfhwastol/carma-messenger-config:${{ github.ref_name }}-chevrolet_tahoe_2018
+          tags: usdotfhwastolcandiate/carma-messenger-config:${{ github.ref_name }}-chevrolet_tahoe_2018
```

Comment:
This tag is the same as the other config images, sounds like it should be detected automatically as well